### DISCUSSION TITLE
Requesting Access to Rosie Software - Research Institute

### DISCUSSION THREAD

**Discussion by mubale-eit** [RosettaCommons/rosetta#196] [ROSIE] [answered]
Hello, I work for Ellison Institute of Technology and would love to use the ROSIE software for non-commercial, research purposes. 
My institution name is Ellison Institute of Technology and my email address is mubale@eit.org. Thanks!

**Comment by lyskov** [comment, marked as answer]
Thank you for letting us know! This should be fixed now. Please feel free to re-open this issue if you still have trouble login-in.
